ext/cfx-ui/src/cfx/common/services/servers/source/listSorter.ts

```diff
@@ -102,3 +102,47 @@ function sortByProperty(servers: Record<string, IListableServerView>, property:
 
   return -dir;
 }
+
+export const sortFilteredList = (servers: Record<string, IListableServerView>, sortedList: string[], config: IServerListConfig): string[] => {
+  const sorters: Array<(a: string, b: string) => number> = [];
+
+  if (shouldPrioritizeServerName(config)) {
```

Comment:
This function can be made into a `getPrioritizedServerName` that'd return the `serverName` if found so we don't have to fetch it from search terms again